### PR TITLE
MWPW-176429: Allow perUnitLabel override

### PR DESCRIPTION
**Pull request by markovicpredrag1209**
Resolves https://jira.corp.adobe.com/browse/MWPW-176429
QA Checklist: https://wiki.corp.adobe.com/display/adobedotcom/M@S+Engineering+QA+Use+Cases

Please do the steps below before submitting your PR for a code review or QA

- [ ] C1. Cover code with Unit Tests
- [ ] C2. Add a Nala test (double check with #fishbags if nala test is needed)
- [ ] C3. Verify all Checks are green (unit tests, nala tests, psi)
- [ ] C4. PR description contains working Test Page link where the feature can be tested
- [ ] C5: you are ready to do a demo from Test Page in PR (bonus: write a working demo script that you'll use on Thursday, you can eventually put in your PR)
- [ ] C.6 read your Jira one more time to validate that you've addressed all AC's and nothing is missing

Test URLs:

- Before: https://main--mas--adobecom.aem.live/
- After: https://mwpw-176429--mas--adobecom.aem.live/
- https://mwpw-174297--milo--adobecom.aem.live/drafts/pedja/ost-legal?mas-io-url=https://14257-merchatscale-markovicpredrag1209.adobeioruntime.net/api/v1/web/MerchAtScale

- This PR requires merging of this pr first: https://github.com/adobecom/milo/pull/4336
MILO PR to allow new field for plans variation:
https://github.com/adobecom/milo/pull/4607

### CODE REVIEW

**Review by 3ch023 — APPROVED** [review]

**Review by npeltier — DISMISSED** [review]

**Comment by afmicka** [comment]
Leaving here as a reminder issues we have discussed offline:
1. perUnit override is enabled for all card types. It should be limited only to Plans cards (plans, plans-students and plans-education)
2. The override does not work at the moment. In the following card there is and override with "per user". However, on the plans page it does not render.
https://mwpw-176429--mas--adobecom.aem.live/studio.html#page=content&path=nala&query=0a12998f-01d6-4ff2-a71e-3fd48999f6b8
Screenshot 2025-07-25 at 16 37 04

https://mwpw-174297--milo--adobecom.aem.live/drafts/nala/features/commerce/plans2?plan=edu_inst&mas-io-url=https://14257-merchatscale-markovicpredrag1209.adobeioruntime.net/api/v1/web/MerchAtScale
Screenshot 2025-07-25 at 16 36 29


cc: @Roycethan

**Review by npeltier — DISMISSED** [review]

File: io/www/test/fragment/pipeline.test.js [hunk 1]
```diff
@@ -266,7 +266,7 @@ describe('pipeline corner cases', () => {
             JSON.stringify({
                 dictionaryId: 'fr_FR_dictionary',
                 translatedId: 'some-fr-fr-fragment',
-                lastModified: 'Tue, 21 Nov 2024 08:00:00 GMT',
+                lastModified: 'Mon, 29 Jul 2025 11:45:13 GMT',
```
[the remark this answers is not in the document]
Comment (markovicpredrag1209) [reply io/www/test/fragment/pipeline.test.js:269]: tests were failing showing me new date, and then i needed to update this one to be after random_old)_date

**Review by 3ch023 — APPROVED** [review]

**Review by npeltier — DISMISSED** [review]

**Review by npeltier — APPROVED** [review]

**Review by 3ch023 — APPROVED** [review]
